sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestFramework/IssueLocationCollector.cs

```diff
@@ -91,10 +107,20 @@ public static bool IsNotIssueLocationLine(string line)
         }
 
         internal /*for testing*/ IEnumerable<IssueLocation> GetIssueLocations(TextLine line)
+        {
+            return GetLocations(line, ISSUE_LOCATION_PATTERN);
+        }
+
+        internal /*for testing*/ IEnumerable<IssueLocation> GetBuildErrorsLocations(TextLine line)
```

Comment:
Add some tests for this method in SonarAnalyzer.UnitTest/TestFramework/IssueLocationCollectorTests